client/less/main.less

```diff
@@ -1175,6 +1175,11 @@ and (max-width : 400px)  {
   font-size: 17px;
 }
 
+// Challange editor CSS for iOS.  Issue 9104.
```

Comment:
Hi,

I wasn't sure exactly what was wanted.

Do you want the editor to expand on the iPad and show the text or it just to be scrollable?

I can change it so it's that way also.  Just that will take a bit longer.

Thank you,
Craig